src/domain/authentication/index.types.d.ts

```diff
@@ -23,17 +26,30 @@ type WithSessionResponse = {
   kratosUserId: UserId
 }
 
+type WithCookieResponse = {
+  cookiesToSendBackToClient: Array<KratosCookie>
```

Comment:
do we send back different type of Cookie, and those changes over time? if it's all the same, we could type of the cookie we need more specifically instead of having an `Array`, ie: 
```
{ 
  cookieSession: CookieSession
  cookieXSS:  CookieXSS
  kratosUserId: UserId
}
```

Reply:
should be just the csrf token and kratos session cookies, unless kratos decides to change the api on us :)